packages/rest/src/explorer.component.ts

```diff
@@ -0,0 +1,50 @@
+// Copyright IBM Corp. 2017. All Rights Reserved.
+// Node module: @loopback/rest
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {Component, CoreBindings} from '@loopback/core';
+import {inject} from '@loopback/context';
+import {RestApplication, RestBindings, Request, Response} from './';
+import {get} from '@loopback/openapi-v3';
+const swaggerUI = require('swagger-ui-dist');
+const path = require('path');
+const readFileSync = require('fs').readFileSync;
+const writeFileSync = require('fs').writeFileSync;
+
+export class ExplorerComponent implements Component {
+  controllers = [ExplorerController];
+
+  constructor(@inject(CoreBindings.APPLICATION_INSTANCE) app: RestApplication) {
+    if (app.config.enableExplorer) {
+      console.log('COMPONENT INIT');
+      const openApiSpecUrl = app.config.openApiSpecUrl || '/openapi.json';
+      const explorerPath = app.config.explorerPath || '/explorer';
+      const explorerLocalPath = swaggerUI.getAbsoluteFSPath();
+      // const explorerIndexFilePath = path.join(explorerLocalPath, 'index.html');
+      // const templateFilePath = path.join(__dirname, 'index.html.tpl');
+      // const templateContent = readFileSync(templateFilePath, 'utf8').replace(
+      //   '<%- openApiSpecUrl %>',
+      //   openApiSpecUrl,
+      // );
+      // writeFileSync(explorerIndexFilePath, templateContent);
+      app.static(explorerPath, explorerLocalPath);
```

Comment:
@bajtos `app.static` seems to be overriding the `ExplorerController `. Any idea what's going on? Or did I misunderstand something?